src/lib/LUA/rx_devLUA.cpp

```diff
@@ -627,6 +667,21 @@ static int event()
   }
   setLuaTextSelectionValue(&luaBindStorage, config.GetBindStorage());
   updateBindModeLabel();
+
+  if (config.GetSourceSysId()==0) 
+  {
+    setLuaUint8Value(&luaSourceSysId,255);
+  } else {
+    setLuaUint8Value(&luaSourceSysId,config.GetSourceSysId());
+  }
+
+  if (config.GetTargetSysId()==0)
+  {
+    setLuaUint8Value(&luaTargetSysId,1);
+  }else{
```

Comment:
(nitpick) Some whitespace would be nice here too, so that its consistent with the above else statement:
`} else {`

Reply:
did all the ws here.